controllers/requestController.js

```diff
@@ -149,6 +151,21 @@ module.exports = {
         });
     },
 
+    release: function (data) {
+        Request.findById(data._id, function(err, request) {
+            if (err) { throw err; }
+
+            var reviewer = data.reviewer;
+            data.reviewer = null;
+            updateRequest(data, request);
+            saveRequest(request);
+
+            // here we bring back the origin reviewer details since other component related on this structure
+            data.oldReviewer = reviewer;
+            notifier.sendSlackPersonalMessage(data.user.email, data.oldReviewer.name, prMessages.released(data), 'orange');
```

Comment:
Build the message before deleting the reviewer, that way you can get rid of `oldReviewer`